<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>921</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>900</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layout" native="true">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>1</y>
      <width>801</width>
      <height>800</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>800</width>
      <height>800</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>500</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QWidget" name="button" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="multiselect">
              <property name="toolTip">
               <string>以进行批量操作</string>
              </property>
              <property name="text">
               <string>多选模式开关</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="showspeed">
              <property name="text">
               <string>流速示意图</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="del">
              <property name="text">
               <string>移除管道</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="add">
              <property name="text">
               <string>添加管道</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QPushButton" name="undo">
              <property name="text">
               <string>撤销</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="redo">
              <property name="text">
               <string>重做</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="reverse">
            <property name="text">
             <string>一键反转</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="resetbtn">
            <property name="text">
             <string>重设输入/出管道</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="checkspeed">
            <property name="text">
             <string>查看管道流速</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="calc">
            <property name="text">
             <string>进行计算</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; font-style:italic;&quot;&gt;输出结果：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="ans1">
              <property name="text">
               <string>1.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="ans2">
              <property name="text">
               <string>2.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="ans3">
              <property name="text">
               <string>3.</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>选择管道编号：</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="speed">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="pipenum">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>流速为：</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>calc</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>calc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>753</x>
     <y>621</y>
    </hint>
    <hint type="destinationlabel">
     <x>859</x>
     <y>595</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>calc()</slot>
 </slots>
</ui>
